### PR TITLE
Issue 7 and Issue 8 - Used flake8 tool and checked pep8 compability and fixed bug then cpf or cnpj are filled with same digit

### PR DESCRIPTION
**Pull request by rodfersou**

### CODE REVIEW

File: pycpfcnpj/calculation.py [hunk 1]
```diff
@@ -1,43 +1,48 @@
 DIVISOR = 11
```
Comment (rodfersou) [inline pycpfcnpj/calculation.py]: Took out TAB characters and used 4 spaces instead

File: pycpfcnpj/cnpj.py [hunk 2]
```diff
@@ -1,28 +1,34 @@
...
 
-	:param cnpj_number: a CNPJ number to be validated.  Only numbers.
-	:type cnpj_number: string
-	:return: Bool -- True for a valid number, False otherwise.
+    if len(cnpj_number) != 14:
+        return False
 
-	"""
+    if len(set(cnpj_number)) == 1:
+        return False
```
Comment (rodfersou) [inline pycpfcnpj/cnpj.py]: This fixes issue 7

File: pycpfcnpj/cpf.py [hunk 3]
```diff
@@ -1,28 +1,34 @@
...
+    """
 
-	"""
+    if len(cpf_number) != 11:
+        return False
 
-	if len(cpf_number) != 11:
-		return False
+    if len(set(cpf_number)) == 1:
+        return False
```
Comment (rodfersou) [inline pycpfcnpj/cpf.py]: This fixes issue 7

File: pycpfcnpj/cnpj.py (hunk 2 above)
Comment (matheuscas) [inline pycpfcnpj/cnpj.py]: Would you mind to put those two conditions in only one conditional?

File: pycpfcnpj/cpf.py (hunk 3 above)
Comment (matheuscas) [inline pycpfcnpj/cpf.py]: Would you mind to put those two conditions in only one conditional?

Comment (matheuscas) [inline pycpfcnpj/cpf.py]: And you must add tests to validate your commit.

**Comment by rodfersou** [comment]
@matheuscas fine I'll do it! thanks

**Comment by rodfersou** [comment]
@matheuscas done!

**Pull request merged**